content/developer/rom/dossier/respondents.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Respondents
+status: draft
+---
+
+* TOC
+{:toc}
+
+Respondents are people that fill out questionnaires. The default respondent is the patient, but there's also profess, parent, second_parent, teacher and caregiver.
+
+- In the future second_parent will be merged into parent.
+- You won't see a profess respondent, since you can't send out to professionals, since there isn't usually just one.
```

Comment:
Prima, als dat met hun via-via is afgesproken. Ik ben er alleen niet zo'n fan van om documentatie individueel naar een klant te communiceren als het in de toekomst algemeen nuttige functionaliteit kan worden, maar mag nu wel zo blijven.